spec.go

```diff
@@ -333,8 +343,14 @@ func createLibcontainerConfig(cgroupName string, spec *specs.LinuxSpec, rspec *s
 		Capabilities: spec.Linux.Capabilities,
 		Readonlyfs:   spec.Root.Readonly,
 		Hostname:     spec.Hostname,
-		Privatefs:    true,
 	}
+
+	if p, exists := mountPropagationMapping[rspec.Linux.RootfsPropagation]; exists {
```

Comment:
I can do config.RootPropagation, exists = ,though, that means I need to define exists in advance.